src/Build.UnitTests/BackEnd/BuildManager_Tests.cs

```diff
@@ -3800,9 +3800,12 @@ public void OutOfProcFileBasedP2PBuildSucceeds()
             }
         }
 
-        /// When a ProjectInstance based BuildRequestData is built out of proc, the node should
+        /// <summary>
+        /// When a <see cref="ProjectInstance"/> based <see cref="BuildRequestData"/> is built out of proc, the node should
         /// not reload it from disk but instead fully utilize the entire translate project instance state
-        /// to do the build
+        /// to do the build.
+        /// </summary>
+        /// <param name="shouldSerializeEntireState">Serializes the entire project instance state if true.</param>
```

Comment:
💡 Consider disabling the rule that requires parameters to have documentation in test projects. I can help complete this in a follow-up pull request if you file an issue for it.

Reply:
@Forgind what are your thoughts on this? If the team chooses to do it, it probably should be done in #7192

Reply:
Sounds reasonable to me. People don't really look at variables in test projects in the same way.